src/cli/commands/force/lightning/lwc/test/ui/configure/mobile.ts

```
@@ -0,0 +1,453 @@
+            'lwc-dev-mobile',
+            Mobile.examples
+        );
+    }
+
+    public static flagsConfig = {
+        ...CommandLineUtils.createFlagConfig(FlagsConfigType.Platform, true),
+        devicename: flags.string({
+            char: 'd',
+            description: messages.getMessage('deviceNameFlagDescription'),
+            required: true,
+            validate: (deviceName) => {
+                if (deviceName && deviceName.trim().length > 0) {
+                    return true;
+                } else {
+                    throw Mobile.createError(
+                        'error:invalidDeviceNameFlagsDescription'
+                    );
+                }
+            }
+        }),
+        output: flags.string({
+            description: messages.getMessage('outputFlagDescription'),
+            required: false,
+            validate: (outputPath) => {
+                if (outputPath && outputPath.trim().length > 0) {
+                    return true;
+                } else {
+                    throw Mobile.createError(
+                        'error:invalidOutputFlagsDescription'
+                    );
+                }
+            }
+        }),
+        testframework: flags.string({
+            description: messages.getMessage('testFrameworkFlagDescription'),
+            required: false,
+            validate: (testFramework) => {
+                const framework = (testFramework ?? '').trim().toLowerCase();
+                if (Mobile.supportedTestFrameworks.includes(framework)) {
+                    return true;
+                } else {
+                    throw Mobile.createError(
+                        'error:invalidTestFrameworkFlagsDescription'
+                    );
+                }
+            }
+        }),
+        bundlepath: flags.string({
+            description: messages.getMessage('bundlePathFlagDescription'),
+            required: false,
+            validate: (bundlePath) => {
```

Should we also validate that the bundle exists at the given path? Might be better to fail fast in this case.

---

Playing devil's advocate for my own comment, you could argue that creating the configuration does not necessary enforce rules upon running it. At a minimum, we may want to log/emit a warning if the bundle path doesn't point to an artifact to install.

---

Given that we're just creating a config file which might be used at a later point, I didn't think we need to error out on these.... but I like the idea of logging warning so I'll update my PR to add that.